dotcom-rendering/src/components/Contributor.tsx

```diff
@@ -50,12 +50,13 @@ const labsBylineStyles = css`
 `;
 
 type Props = {
-	byline: string;
+	byline?: string;
+	source?: string;
```

Comment:
This creates four possible states, what do they each mean?

|                | With byline | Without byline |
|----------------|-------------|----------------|
| With source    | A.          | C.             | 
| Without source | B.          | D.             |